### PR TITLE
feat: import sample volunteers into Keycloak

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

File: docs/coding.md [hunk 1]
```diff
@@ -76,6 +76,26 @@ Follow these instructions to set up Keycloak for Authentication/Authorization.
...
+This command will create the following usernames:
+
+```log
+hzaub	
+kudi
+msash
+tgers
+tmaure
+ukon
+wtrocki
```
Comment (wtrocki) [inline docs/coding.md:93]: LOL - this is accidental name but like it. I would add yours as well :D

File: platform/server/integrations/keycloak/initKeycloak.js [hunk 2]
```diff
@@ -33,16 +33,110 @@ const PUBLIC_CLIENT_NAME = 'open-volunteer-platform-public'
 const BEARER_CLIENT_NAME = 'open-volunteer-platform-bearer'
 let PUBLIC_CLIENT
 
+let demoUsers = process.env.USE_DEMO_DATA !== 'true' ? []: [
+  {
+    name: 'msash',
```
Comment (wtrocki) [inline platform/server/integrations/keycloak/initKeycloak.js:38]: I think we need some designated admin here. 1 or two would be good and they cannot have actions assigned

Comment (machi1990) [reply platform/server/integrations/keycloak/initKeycloak.js:38]: I was thinking just about that. I'll rename the default `admin:admin` to `ovp-admin:ovp-admin`, WDYT?

**Comment by wtrocki** [comment]
ovpadmin sound better for me :)

File: docs/coding.md (hunk 1 above)
Comment (machi1990) [reply docs/coding.md:93]: :-)

File: platform/server/integrations/keycloak/initKeycloak.js (hunk 2 above)
Comment (machi1990) [reply platform/server/integrations/keycloak/initKeycloak.js:38]: Done through out.

File: platform/server/integrations/keycloak/initKeycloak.js [hunk 3]
```diff
@@ -259,12 +354,15 @@ async function getClientRoles(client) {
   return res.data
 }
 
-async function createUser(name, password) {
+async function createUser({name, password, email, firstName, lastName}) {
```
Comment (wtrocki) [inline platform/server/integrations/keycloak/initKeycloak.js:357]: Love it!

File: platform/server/integrations/keycloak/initKeycloak.js [hunk 4]
```diff
@@ -112,7 +207,7 @@ async function prepareKeycloak() {
     for (let user of users) {
       // Create a new user
       console.log(`creating user ${user.name} with password ${user.password}`)
-      const userIdUrl = await createUser(user.name, user.password)
+      const userIdUrl = await createUser(user)
```
Comment (wtrocki) [inline platform/server/integrations/keycloak/initKeycloak.js:210]: Actually we might backport this changes to datasync-starter.
Amazing job!

**Pull request merged**